embedded/EscapeROOM/radio_morse_panel/radio_morse_panel.c

```diff
@@ -10,7 +12,11 @@
 #define RED_LED PD3
 #define GREEN_LED PD4
 
+static volatile uint32_t msFromReset;
+
 const char morsePhraseToPlay[] = "472";
+const char keysToPress[] = "357";
+#define PASS_LEN sizeof(keysToPress) / sizeof(keysToPress[0])
```

Comment:
I'm in the mood of renaming this evening.
keysToPress -> PASSWORD? It is constant in your code.
If keysToPress is zero-terminated string, then do you need PASS_LEN? Maybe use MAX_PASS_LEN to set buffer sizes?

Reply:
Good will use this.
Actually i like Your comments a lot. They are very significant for me. I am actually learning and improving my skill from them. Thank you.